src/tallies/tally_header.F90

```diff
@@ -354,6 +354,17 @@ function tally_set_filters(this, filter_indices) result(err)
         j = FILTER_AZIMUTHAL
       type is (EnergyFunctionFilter)
         j = FILTER_ENERGYFUNCTION
+      type is (LegendreFilter)
+        j = FILTER_LEGENDRE
+        this % estimator = ESTIMATOR_ANALOG
+      type is (SphericalHarmonicsFilter)
+        j = FILTER_SPH_HARMONICS
```

Comment:
Could we set a default estimator based on the cosine_type ?
I'm thinking collision for flux, and analog for scatter

Reply:
Good suggestion; I set it to analog for scatter and left the default otherwise (since the filter can work with tracklength estimators if `p%mu` is not required)